cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -378,6 +385,8 @@ static DataprocConf create(Map<String, String> properties, @Nullable SSHPublicKe
                                                                                      "true"));
     boolean stackdriverMonitoringEnabled = Boolean.parseBoolean(properties.getOrDefault(STACKDRIVER_MONITORING_ENABLED,
                                                                                         "true"));
+    boolean componentGatewayEnabled = Boolean.parseBoolean(properties.getOrDefault(COMPONENT_GATEWAY_ENABLED,
```

Comment:
No need to call `getOrDefault`. just call `get` is ok. `Boolean.parseBoolean` will return `false` for `null`.

Reply:
Done.